test/functional/specs/Personalization/C28762.js

```diff
@@ -0,0 +1,107 @@
+import { t } from "testcafe";
+import createNetworkLogger from "../../helpers/networkLogger";
+import { responseStatus } from "../../helpers/assertions/index";
+import createFixture from "../../helpers/createFixture";
+import {
+  compose,
+  orgMainConfigMain,
+  debugEnabled
+} from "../../helpers/constants/configParts";
+import getResponseBody from "../../helpers/networkLogger/getResponseBody";
+import createResponse from "../../helpers/createResponse";
+import { TEST_PAGE as TEST_PAGE_URL } from "../../helpers/constants/url";
+import createAlloyProxy from "../../helpers/createAlloyProxy";
+
+const networkLogger = createNetworkLogger();
+const config = compose(orgMainConfigMain, debugEnabled);
+const PAGE_WIDE_SCOPE = "__view__";
+const FORM_BASED_SCOPE = "form-based-scope";
+
+const DEFAULT_CONTENT_ITEM = {
+  id: "0",
+  schema: "https://ns.adobe.com/personalization/default-content-item",
+  meta: {
+    "activity.id": "139738",
+    "activity.name": "Functional: C28762",
+    "experience.id": "0",
+    "offer.id": "0",
+    "offer.name": "Default Content"
+  }
+};
+
+const MEASUREMENT_ITEM = {
+  id: "139738",
+  schema: "https://ns.adobe.com/personalization/measurement",
+  data: {
+    format: "application/vnd.adobe.target.metric",
+    type: "click"
+  }
+};
+
+createFixture({
+  title: "C28762: Merged metric propositions should be delivered",
+  url: `${TEST_PAGE_URL}?test=C28762`,
+  requestHooks: [networkLogger.edgeEndpointLogs]
+});
+
+test.meta({
+  ID: "C28762",
+  SEVERITY: "P0",
+  TEST_RUN: "Regression"
+});
+
+test("Test C28762: Merged metric propositions should be delivered", async () => {
```

Comment:
You need to add a test to https://testrail.corp.adobe.com/index.php?/suites/view/25 and use that number here.

Reply:
Hopefully @shammowla will provide me with access to TestRail today and I'll add it

Reply:
@jonsnyder I finally got access to TestRail today and renamed both the default content and merged metrics functional tests, please take another look. Thanks!